packages/react-components/react-calendar-compat/src/components/CalendarMonth/CalendarMonth.tsx

```diff
@@ -242,7 +245,7 @@ export const CalendarMonth: React.FunctionComponent<CalendarMonthProps> = props
             }
             type="button"
           >
-            {dir === 'ltr' ? <ArrowDownRegular /> : <ArrowUpRegular />}
+            {dir === 'rtl' ? downNavigationIcon : upNavigationIcon}
```

Comment:
turns out these were flipped, so as a side fix, CalendarMonth now works properly in RTL

Reply:
Looking at the snapshots and the deployed storybook, I think that they were right before, weren't they?